src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.js

```diff
@@ -186,10 +186,28 @@ uiModules.get('apps/management')
       });
   };
 
+  this.isTimeBased = () => {
+    if (!this.formValues.timeFieldOption) {
+      // if they haven't choosen a time field, assume they will
+      return true;
+    }
+
+    // if timeFieldOption has a fieldName it's a time field, otherwise
+    // it's a way to opt-out of the time field or an indication that there
+    // are no fields available
+    return !!this.formValues.timeFieldOption.fieldName;
+  };
+
   this.canExpandIndices = () => {
-    // to maximize performance in the digest cycle, move from the least
-    // expensive operation to most
-    return !this.formValues.nameIsPattern && _.includes(this.formValues.name, '*');
+    return (
+      this.isTimeBased() &&
+        !this.formValues.nameIsPattern &&
+        _.includes(this.formValues.name, '*')
```

Comment:
Out of scope for this PR, but isn't `this.formValues.name` a string? If it's a string, we probably don't need to use the lodash `.includes(` method which is normally used on collections.